pantheon-bundle/frontend/src/app/moduleDisplay.tsx

```diff
@@ -143,6 +144,7 @@ class ModuleDisplay extends Component<any, any, any> {
                         <Card>
                             <Versions
                                 modulePath={this.state.modulePath}
+                                type="module"
```

Comment:
please create an entry for module in pantheon-bundle/frontend/src/app/Constants.tsx